codes/dataops/augmentations.py

```diff
@@ -502,6 +502,20 @@ def random_crop_pairs(img_HR, img_LR, HR_size, scale):
     
     return img_HR, img_LR
 
+
+def random_crop_dvd(img_top, img_bot, img_in, patch_size):
```

Comment:
Should reuse https://github.com/victorca25/BasicSR/blob/6b2317617c957e0805bcefbe614428e6f36fda00/codes/data/Vid_dataset.py#L487 . May be better to convert into a reusable class